src/twix.coffee

```diff
@@ -515,3 +515,6 @@ if  typeof(define) == "function"
   define "twix", ["moment"], (moment) -> makeTwix(moment)
 
 @Twix = makeTwix(@moment) if @moment?
+
+# Also checks globals (Meteor)
+@Twix = makeTwix(moment) if moment?
```

Comment:
from my side it's fine this way...

The thing is with Metor you work mostly with globals, so after having included the [Moment package](https://atmospherejs.com/momentjs/moment) you don't see it as `@moment` but only as a true global `moment`

...then I use `@Twix = ...` just to be able to pick up `Twix` on the `locale.js` file (see [this line](https://github.com/icambron/twix.js/pull/55/files#diff-8b5cad59fb44303bb6585c92e33f7192R34) where I check for `@Twix && moment`) but then I delete `@Twix` straight after to clean up the `this` object...

Does this makes any sense to you?
I don't pretend you to enter the Meteor world ;-)

this was the only way I found to get it working both within and outside Meteor, with all tests passing!